### PR TITLE
[NEW] Add message action menu item to copy permalink from tapped message

### LINKED ISSUE #1803: [NEW] Support "Hyperlink" of the message to be copied
## Description

If someone long press a message, there should be an option to copy the hyperlink of it.

### PR DESCRIPTION
**Pull request by leonardoaramaki**
@RocketChat/android

Closes #1803

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -213,6 +213,7 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="action_msg_share">Share</string>
     <string name="action_title_editing">Editing Message</string>
     <string name="action_msg_add_reaction">Add reaction</string>
+    <string name="action_msg_permalink" translatable="false">Permalink</string>
```
Comment (rafaelks) [inline app/src/main/res/values/strings.xml]: "Copy Permalink"

**Review by rafaelks — DISMISSED** [review]
Missing other languages...

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (leonardoaramaki) [reply app/src/main/res/values/strings.xml]: Will the android client be different than the web?

Comment (rafaelks) [reply app/src/main/res/values/strings.xml]: @leonardoaramaki Yes... makes much more sense having the action (like all the other actions here).

Comment (leonardoaramaki) [reply app/src/main/res/values/strings.xml]: I usually try to follow the web client. On the web there's no such pattern. I.e. there you have "Star message" and just "Permalink"...

**Review by luciofm — APPROVED** [review]

**Pull request merged**